src/Client/wwwroot/index.html

```diff
@@ -13,16 +13,22 @@
 </head>
 
 <body>
-    <div id="app">Loading, be patient...</div>
+    <div id="app">
+        <div style="position: absolute; top: 30vh; width: 100%; text-align: center; font-family: 'Roboto';">
```

Comment:
This is fine, I suppose. We're not gonna touch index.html latery anyway and it's not that bloated. We might change this during refactor 😊